go_test.mod

```diff
@@ -14,6 +14,7 @@ require (
 	github.com/klauspost/compress v1.15.9 // indirect
 	github.com/minio/highwayhash v1.0.2 // indirect
 	github.com/nats-io/jwt/v2 v2.3.0 // indirect
+	go.uber.org/automaxprocs v1.5.1 // indirect
```

Comment:
Weird.. I check out your branch and then did a `go mod tidy -modfile=go_test.mod` and that was removed, and go.sum is updated, so I would recommend you do that.

Reply:
Oh. yeah weird that it was brought in. Thanks for the catch